src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -4384,18 +4313,17 @@ namespace SystemAvailabilityManager {
             }
 
             // Check simple airflow object
-            if (state.dataSystemAvailabilityManager->HybridVentSysAvailMgrData(SysAvailNum).SimpleControlTypeSchedPtr > 0 &&
-                state.dataSystemAvailabilityManager->HybridVentSysAvailMgrData(SysAvailNum).VentilationPtr > 0) {
-                if (state.dataSystemAvailabilityManager->HybridVentSysAvailMgrData(SysAvailNum).ActualZoneNum !=
-                    state.dataHeatBal->Ventilation(state.dataSystemAvailabilityManager->HybridVentSysAvailMgrData(SysAvailNum).VentilationPtr)
+            if (hybridVentMgr.SimpleControlTypeSchedPtr > 0 && hybridVentMgr.VentilationPtr > 0) {
+                if (hybridVentMgr.ActualZoneNum !=
+                    state.dataHeatBal->Ventilation(hybridVentMgr.VentilationPtr)
                         .ZonePtr) {
-                    ShowSevereError(state, std::string{RoutineName} + cCurrentModuleObject + "=\"" + state.dataIPShortCut->cAlphaArgs(1) + "\"");
+                    ShowSevereError(state, format("{}{}=\"{}\"", RoutineName, cCurrentModuleObject, state.dataIPShortCut->cAlphaArgs(1)));
```

Comment:
Changed `ShowSevereError` calls to use `format` when `cCurrentModuleObject` was changed to `string_view`.